example-conf/mesos-dns/mesos-site.xml

```diff
@@ -43,7 +43,7 @@
 
   <property>
     <name>mesos.hdfs.zkfc.ha.zookeeper.quorum</name>
-    <value>master0.mesos:2181,master1.mesos:2181,master2.mesos:2181</value>
```

Comment:
That's right. HDFS is by its nature fault tolerant and we automate
different failure scenarios through a state machine in our scheduler. We
use the data replication inherent in HDFS by launching multiple data nodes
and name nodes. We will be integrating with
https://issues.apache.org/jira/browse/MESOS-1554 for better management of
resources and further resilience. I would love to discuss further as
needed, message me at elizabeth@mesosphere.io to continue the discussion.

On Fri, Mar 27, 2015 at 3:56 PM, hi10-artist <notifications@github.com>
wrote:

> In example-conf/mesos-dns/mesos-site.xml
> <https://github.com/mesosphere/hdfs/pull/113#discussion_r27337000>:
>
> > @@ -43,7 +43,7 @@
> >
> >    <property>
> >      <name>mesos.hdfs.zkfc.ha.zookeeper.quorum</name>
> > -    <value>master0.mesos:2181,master1.mesos:2181,master2.mesos:2181</value>
>
> Hey guys,
> Sorry for poking you here (couldn't find other channel/contact point).
> Hope that's okay?
> Thanks for working on this. Skimming through it, looks like you guys are
> working on starting the HDFS processes on mesos. Could you please tell how
> are you working on to providing resilience in case of slave failure (in
> case if a mesos slave task dies, etc), given that
> https://issues.apache.org/jira/browse/MESOS-1554 is still in progress.
> Or, does this work stand on top of the 1554?
> Thanks,
> Himanshu
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/hdfs/pull/113/files#r27337000>.
>